contracts/integrations/DIVAProtocolMock.sol

```diff
@@ -39,60 +37,42 @@ contract DIVAProtocolMock {
 
     // Add some fake pools
     constructor() {
```

Comment:
Did you mean to keep the commented out code?

Reply:
nah, ty